vignettes/adsl.Rmd

```diff
@@ -716,6 +717,36 @@ dataset_vignette(
 )
 ```
 
+### Derive Randomization Date (`RANDDT`) {#randomization_date}
+
+The function `derive_vars_merged_dt()` can be used to derive randomization date 
+variable.
+
+By default, the function will map DS records where DSDECOD="RANDOMIZED". To derive 
+the Randomization Date (`RANDDT`), the call would be:
+
+```{r eval=TRUE}
+adsl <- adsl %>%
+  derive_vars_merged_dt(
+    dataset_add = ds,
+    filter_add = DSDECOD == "RANDOMIZED",
+    by_vars = vars(STUDYID, USUBJID),
+    new_vars_prefix = "RAND",
+    dtc = DSSTDTC,
+    flag_imputation = FALSE
+  )
+```
+
+
+This call would return the input dataset with the column `RANDDT` and by default
+the associated date imputation flag (`RANDDTF`) are added. If the imputation flag
+is not required, the user must set the argument `flag_imputation` to FALSE.
+
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(adsl, display_vars = vars(USUBJID, ARM, ACTARM, RANDDT))
```

Comment:
delete "ARM, ACTARM, "

Reply:
Reason I kept ARM and ACTARM - to explain `NA` values, For Screen Failure records the `RANDDT` value is `NA`. Let me know know your views on it.

Reply:
do we have any screen fails in the test data? i know there's a discussion on this in the slack channel but personally i never saw screen fail data via SDTM. i would say ok to drop these columns but happy for you to make the call

Reply:
> do we have any screen fails in the test data? i know there's a discussion on this in the slack channel but personally i never saw screen fail data via SDTM. i would say ok to drop these columns but happy for you to make the call

![image](https://user-images.githubusercontent.com/90273880/174307529-cb57a98a-4f82-46c9-b118-e921e4cb9791.png)
FYI- Yes we do have Screen Failure Records in test data. I have updated to remove ARM, ACTARM.